.storybook/pages/WireframeDemo/WireframeDemo.module.css

```diff
@@ -0,0 +1,191 @@
+@import '../../../src/design-tokens/mixins.css';
+
+/*------------------------------------*\
+    # WIREFRAME DEMO
+\*------------------------------------*/
+
+/**
+ * WireframeDemo
+ */
```

Comment:
nit/q: are we still adding comments like this or is it ok to just remove them?

Reply:
This top one's pretty pointless; I can remove it. I'll leave the ones below though because I like that they separate the styles into sections. I suppose that's a signal that I should break it out into separate components in separate files, but I don't want to bloat the directory structure with subcomponents for these two demos.